### DISCUSSION TITLE
Could a custom driver based on a postgrest API could work?

### DISCUSSION THREAD

**Discussion by tigitz** [mikro-orm/mikro-orm#5062] [Ideas]
Might be a crazy idea but using  https://github.com/supabase/postgrest-js I find myself reinventing some kind of an ORM.

I hydrate my object from a GET call, change object values and eventually send a POST call to persist my changes.

Postgrest also handles relations etc. It's basically a subset of the most common sql operations over http.

Of course migrations etc wouldn't work at first (although, all the necessary operations could be exposed through an API)

Am I missing an obvious blocker?

**Comment by B4nan** [comment]
Take a look how mongo driver is implemented, maybe it could work in a similar way.
